src/schema/comments.ts

```diff
@@ -297,99 +296,59 @@ export const resolvers: IResolvers<any, Context> = {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   Query: traceResolverObject<any, any>({
     postComments: async (
-      source,
+      _,
       args: GQLPostCommentsArgs,
       ctx,
       info,
     ): Promise<Connection<GQLComment>> => {
-      const page = commentsPageGenerator.connArgsToPage(args);
-      return graphorm.queryPaginated(
-        ctx,
-        info,
-        (nodeSize) => commentsPageGenerator.hasPreviousPage(page, nodeSize),
-        (nodeSize) => commentsPageGenerator.hasNextPage(page, nodeSize),
-        (node, index) =>
-          commentsPageGenerator.nodeToCursor(page, args, node, index),
-        (builder) => {
-          builder.queryBuilder = builder.queryBuilder
-            .andWhere(`${builder.alias}.postId = :postId`, {
-              postId: args.postId,
-            })
-            .andWhere(`${builder.alias}.parentId is null`)
-            .orderBy(`${builder.alias}."createdAt"`)
-            .limit(page.limit);
-          if (page.timestamp) {
-            builder.queryBuilder = builder.queryBuilder.andWhere(
-              `${builder.alias}."createdAt" > :timestamp`,
-              { timestamp: page.timestamp },
-            );
-          }
-          return builder;
-        },
-      );
+      const pageGenerator = new GQLPageGenerator<GQLComment>();
```

Comment:
We can create all these instances of the page generator outside the resolver. What do you think?

Reply:
I think that is a good idea, to have less memory footprint. Then we let's allow passing down of config on a much later stage of the generator's life.